docs/source/markdown/podman-image-prune.1.md

```diff
@@ -11,6 +11,9 @@ podman-image-prune - Remove all unused images from the local store
 you can delete all unused images.  Unused images are dangling images as well as any image that
 does not have any containers based on it.
 
+It skips the cache images i.e. intermediate images build during image building.
+Cache images are deleted along with main image deletion.
```

Comment:
deletion -> deletion when using the podman rmi command.

and if you go with that, we probably ought to add a pointer to podman rmi.